elyra/pipeline/pipeline_definition.py

```diff
@@ -638,14 +612,18 @@ def get_canvas_properties_from_template(package_name: str, template_name: str) -
         return json.loads(output)
 
     @staticmethod
-    def get_key_value_pair_properties(canvas_properties) -> List[str]:
+    def get_pipeline_kv_pair_properties() -> List[str]:
```

Comment:
Since this method is on `PipelineDefintion`, I think we can safely assume the kv properties are relative to the pipeline and drop that from the method name as well, leaving...
```python
 def get_kv_properties() -> List[str]:
```